stackslib/src/net/api/getinfo.rs

```diff
@@ -130,6 +131,27 @@ impl RPCPeerInfoData {
         let public_key_buf = StacksPublicKeyBuffer::from_public_key(&public_key);
         let public_key_hash = Hash160::from_node_public_key(&public_key);
         let stackerdb_contract_ids = network.get_local_peer().stacker_dbs.clone();
+        // let is_fully_synced = if network.work_state == PeerNetworkWorkState::BlockDownload || network.work_state == PeerNetworkWorkState::BlockInvSync {
+        //     false
+        // } else {
+        //     true
+        // };
+
+        let is_fully_synced = if let Some(block_downloader) = &network.block_downloader {
+            !block_downloader.is_initial_download()
```

Comment:
I'm assuming this is not the right approach as on Linux after starting a sync on top of an existent one, it was displaying `is_fully_synced true`. Is there an easier way to test this implementation?

Reply:
This is not the right approach.  You need to serve the value of `ibd` that gets passed into `PeerNetwork::run()`

Reply:
I've updated it by referencing the `ibd` that is used as value. I also though about saving it in PeerNetwork but that would have mean the source of truth is again split in 2 as until the `PeerNetwork::run()` is executed it has a default value. https://github.com/stacks-network/stacks-core/compare/develop...ASuciuX:stacks-core:feat/is-synced-v2-info